hr_public_holidays/__openerp__.py

```diff
@@ -23,7 +23,8 @@
     'name': 'Public Holidays',
     'version': '1.0',
     'category': 'Generic Modules/Human Resources',
-    'author': "Michael Telahun Makonnen <mmakonnen@gmail.com>,Odoo Community Association (OCA)",
+    'author': "Michael Telahun Makonnen <mmakonnen@gmail.com>,"
+              "Odoo Community Association (OCA)",
```

Comment:
Put yourself in the authors, add README.rst for description and detail it like in the template from the https://github.com/OCA/maintainer-tools